configureBuild.sh

```diff
@@ -237,15 +236,17 @@ processArgumentsforSpecificArchitectures() {
       jvm_variant=server,client
       make_args_for_any_platform="DEBUG_BINARIES=true images legacy-jre-image"
     fi
-    configure_args_for_any_platform="--with-jobs=${BUILD_CONFIG[NUM_PROCESSORS]}"
+    if [[ ${BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]:-""} != *"--with-jobs"* ]]; then
```

Comment:
Is this change supposed to be in this PR?

Reply:
Yes. It removes the need for the lowercase configure_args_for_any_platform variable, and has the additional benefit of ensuring we have a --with-jobs parameter set here even if one was not set by the user (which seems to be the point of this code, as per the removed line below).

```
BUILD_CONFIG[CONFIGURE_ARGS_FOR_ANY_PLATFORM]=${BUILD_CONFIG[CONFIGURE_ARGS_FOR_ANY_PLATFORM]:-$configure_args_for_any_platform}
```

Currently it looks like the only reason this works is because we don't set CONFIGURE_ARGS_FOR_ANY_PLATFORM anywhere else, which means the user always has at least one --with-jobs arg. But now we're removing CONFIGURE_ARGS_FOR_ANY_PLATFORM, the presence of any text in USER_SUPPLIED_CONFIGURE_ARGS will prevent the required --with-jobs argument being added even if you don't have one already, which I believe to be bad news on armv71.